lta/config.py

```diff
@@ -2,15 +2,19 @@
 
 import os
 
+NA = (0,) # use a tuple so 'is' operator works correctly
 
 def from_environment(keys):
```

Comment:
I was really impressed that no additional unit tests were required to keep 100% coverage and add support for dictionary input.